### ISSUE TITLE
Brightness Controls are not working

### ISSUE THREAD

**Issue by deepanshpandey** [yusufklncc/Lenovo-Legion-5-Hackintosh#23]
![Screenshot 2023-09-09 at 7 52 31 PM](https://github.com/yusufklncc/Lenovo-Legion-5-Hackintosh/assets/60426924/a1905da5-b2c2-4a3f-8eaf-705bae2e5ea0)
I installed mac os ventura using the guide below and can't control brightness it is fixed at one specific level

**Comment by deepanshpandey** [comment]
If I can provide anything do tell me so I can grab it and send
my display is a 1920x1080 panel with 144hz refresh rate
![Screenshot 2023-09-09 at 7 49 18 PM](https://github.com/yusufklncc/Lenovo-Legion-5-Hackintosh/assets/60426924/eea77269-1f18-46f4-b79c-37f5817ee7d7)
![Screenshot 2023-09-09 at 7 51 11 PM](https://github.com/yusufklncc/Lenovo-Legion-5-Hackintosh/assets/60426924/e1971f07-0547-435f-a12f-c1a07010339f)

**Comment by rafysp** [comment]
Same here, did you have found the solution?

**Comment by deepanshpandey** [comment]
> Same here, did you have found the solution?

nope, I switched back to windows later as latest macos was also not booting...

**Comment by rafysp** [comment]
Maybe it really isn't possible to change the screen brightness natively because the type of screen uses DC dimming. I'm using an application called Lunar to adjust the screen brightness. You need to update kext to using latest macos, right now i'm on sonoma 14.2.1

**Comment by rafysp** [comment]
> > Same here, did you have found the solution?
> 
> nope, I switched back to windows later as latest macos was also not booting...

Maybe it really isn't possible to change the screen brightness natively because the type of screen uses DC dimming. I'm using an application called Lunar to adjust the screen brightness. You need to update kext to using latest macos, right now i'm on sonoma 14.2.1

**Comment by deepanshpandey** [comment]
> > > Same here, did you have found the solution?
> > 
> > 
> > nope, I switched back to windows later as latest macos was also not booting...
> 
> Maybe it really isn't possible to change the screen brightness natively because the type of screen uses DC dimming. I'm using an application called Lunar to adjust the screen brightness. You need to update kext to using latest macos, right now i'm on sonoma 14.2.1

will try this for sure

**Comment by rafysp** [comment]
> > > > Same here, did you have found the solution?
> > > 
> > > 
> > > nope, I switched back to windows later as latest macos was also not booting...
> > 
> > 
> > Maybe it really isn't possible to change the screen brightness natively because the type of screen uses DC dimming. I'm using an application called Lunar to adjust the screen brightness. You need to update kext to using latest macos, right now i'm on sonoma 14.2.1
> 
> will try this for sure

Suddenly, I can change the brightness natively! Previously, I wanted to patch the framebuffer for HDMI, so I booted into Windows to change nvidia physx and set the BIOS from dynamic graphics to discrete graphics. Then, I reverted everything to the original settings, and when I booted into macOS, surprisingly, the brightness control worked now.

**Comment by deepanshpandey** [comment]
now I can say that I jealous of you 😀

**Comment by rafysp** [comment]
> now I can say that I jealous of you 😀

you can download my EFI on my github and test it by yourself. i change the smbios to macbookpro16,3 because macbookpro16,4 expect an dGPU, maybe that's what made my brightness control work.

**Comment by deepanshpandey** [comment]
starred it...
will install macos again this weekend 
thanks 🔥
